src/skin/legacyskinparser.cpp

```diff
@@ -1081,6 +1084,24 @@ QWidget* LegacySkinParser::parseStarRating(const QDomElement& node) {
 
 
 
+QWidget* LegacySkinParser::parseRateDisplay(const QDomElement& node) {
+    QString group = lookupNodeGroup(node);
+    QColor c(255,255,255);
```

Comment:
From what I can tell all that color code is obsolete. (here and below in parseNumberRate)
The parsing code can be as compact [that for parseNumberPos](https://github.com/mixxxdj/mixxx/pull/2942/files#diff-c2c84548a4801a7ca8649f1095e0bf36R1127)

Unrelated to your changes but a good opportunity to clean it up in this PR IMO.

Reply:
In another PR I wanted to clean another code, other were not happy with this, therefore I will just leave it as it is, but will clean my function

Reply:
it'll be okay if you remove if from this and the close neighbour (and related widget) NumberRate